tests/scripts/basic-build-test.sh

```diff
@@ -39,6 +39,10 @@ fi
 CONFIG_H='include/mbedtls/config.h'
 CONFIG_BAK="$CONFIG_H.bak"
 
+# Step 0 - print build environment info
+scripts/print_dev_env.sh
```

Comment:
This script only gives you dump of useful information about the machine's setup, but perhaps we do need something that actually raises an error, or perhaps skips over tests because the required utilities are not available?